README.md

```diff
@@ -145,11 +145,23 @@ Type: `String[]`<br>
 Default: `[]`
 
 ### timeout
-Timeout for step definitions.
+Timeout in milliseconds for step definitions.
 
 Type: `Number`<br>
 Default: `30000`
 
+### ignoreUndefinedDefinitions
```

Comment:
I don't see `ignoreUndefinedDefinitions` being used anywhere. It is also not an option in the cucumber.js framework as far as I can see.